src/test/java/org/opensearch/knn/index/VectorDataTypeIT.java

```diff
@@ -263,29 +263,6 @@ public void testByteVectorDataTypeWithNmslibEngine() {
         assertTrue(ex.getMessage().contains("is not supported for vector data type"));
     }
 
-    @SneakyThrows
-    public void testByteVectorDataTypeWithLegacyFieldMapperKnnIndexSetting() {
```

Comment:
Just checking if this case is covered when engine is explicitly NMSLIB

Reply:
Yes, it is covered explicitly for nmslib engine